### PR TITLE
add a `user/<username>` route for reviews

### PR DESCRIPTION
**Pull request by jdhollen**
gonna use this to make sure i make your view not ugly in #5691 

**Related issues**: N/A

### CODE REVIEW

**Review by siggisim — APPROVED** [review]

File: enterprise/app/review/BUILD [hunk 1]
```diff
@@ -8,7 +8,6 @@ ts_library(
     name = "review",
     srcs = ["review.tsx"],
     deps = [
-        "//app/auth:user",
```
Comment (jdhollen) [inline enterprise/app/review/BUILD:11]: ignore this -- test

File: enterprise/app/review/BUILD [hunk 2]
```diff
@@ -21,7 +20,6 @@ ts_library(
     name = "review_list",
     srcs = ["review_list.tsx"],
     deps = [
-        "//app/auth:user",
         "//app/format",
```
Comment (jdhollen) [inline enterprise/app/review/BUILD:25]: ignore this -- test

File: enterprise/app/review/BUILD [hunk 3]
```diff
@@ -37,7 +35,6 @@ ts_library(
     name = "view_pull_request",
     srcs = ["view_pull_request.tsx"],
     deps = [
-        "//app/auth:user",
         "//app/errors:error_service",
```
Comment (jdhollen) [inline enterprise/app/review/BUILD:38]: last one

File: enterprise/app/review/BUILD (hunk 2 above)
Comment (jdhollen) [reply enterprise/app/review/BUILD:25]: this is a reply

File: enterprise/app/review/BUILD [hunk 4]
```diff
@@ -8,7 +8,6 @@ ts_library(
     name = "review",
     srcs = ["review.tsx"],
     deps = [
-        "//app/auth:user",
         "//app/router",
```
Comment (jdhollen) [inline enterprise/app/review/BUILD:12]: trying to post as a bot.

File: enterprise/app/review/BUILD (hunk 2 above)
Comment (jdhollen) [reply enterprise/app/review/BUILD:25]: replying inline

File: enterprise/app/review/BUILD (hunk 4 above)
Comment (jdhollen) [reply enterprise/app/review/BUILD:12]: Unresolving.

Comment (jdhollen) [reply enterprise/app/review/BUILD:12]: Let's try that again.

Comment (jdhollen) [reply enterprise/app/review/BUILD:12]: Aaand resolving.

Comment (jdhollen) [reply enterprise/app/review/BUILD:12]: make sure this doesn't break anything (sorry if you're reading this)

File: enterprise/app/review/BUILD (hunk 1 above)
Comment (jdhollen) [reply enterprise/app/review/BUILD:11]: Done.

Comment (jdhollen) [reply enterprise/app/review/BUILD:11]: Acknowledged.

Comment (jdhollen) [reply enterprise/app/review/BUILD:11]: Unresolved

Comment (jdhollen) [reply enterprise/app/review/BUILD:11]: Acknowledged.

**Pull request merged**